apis/control/v1alpha1/k8ssandratask_types.go

```diff
@@ -199,3 +175,47 @@ func getConditionStatus(s cassapi.CassandraTaskStatus, conditionType cassapi.Job
 func init() {
 	SchemeBuilder.Register(&K8ssandraTask{}, &K8ssandraTaskList{})
 }
+
+// getCondition converts a primitive boolean into a k8s ConditionStatus
+func getCondition(condition bool) corev1.ConditionStatus {
+	if condition {
+		return corev1.ConditionTrue
+	}
+	return corev1.ConditionFalse
+}
+
+// getGlobalStatus returns an aggregate status of the Datacenters task statuses
+func getGlobalStatus(t K8ssandraTaskStatus) globalStatus {
```

Comment:
Maybe we could make this a method since it's kind of the same level as `RefreshGlobalStatus`:
```go
func (t * K8ssandraTask) getGlobalStatus() globalStatus ...
```